module/dal/control/src/main/java/org/openbase/bco/dal/control/action/ActionImpl.java

```diff
@@ -80,6 +80,13 @@ public class ActionImpl implements SchedulableAction {
     private ServiceDescription serviceDescription;
     private volatile Future<ActionDescription> actionTask;
 
+    /**
+     * This flag ist required to handle the case when the action task was
+     * initiated but never started because of any interruption.
+     * In this case the flag is false but the actionTask non null.
+     */
+    private volatile Boolean actionTaskRunning;
```

Comment:
Is there any reason for not initializing this to `false`? I know that it is always initialized before the `actionTask` and thus the `isActionTaskFinished` call cannot produce a null pointer. But I think this unnecessarily provokes a null pointer if the code is changed in the future.